tmva/tmva/inc/TMVA/DNN/CNN/ConvLayer.h

```diff
@@ -46,21 +46,35 @@ class TConvLayer : public VGeneralLayer<Architecture_t> {
    using Scalar_t = typename Architecture_t::Scalar_t;
 
 private:
+   bool inline isInteger(Scalar_t x) const { return x == floor(x); }
+
+   /* Calculate the output dimension of the convolutional layer */
+   size_t calculateDimension(int imgDim, int fltDim, int padding, int stride);
```

Comment:
for consistency it is maybe better to have the parameters defined as size_t

Reply:
You are right I will make the change ASAP